core/ui/RootStylesheet.tid

```diff
@@ -0,0 +1,19 @@
+title: $:/core/ui/RootStylesheet
+code-body: yes
+
+\import [subfilter{$:/core/config/GlobalImportFilter}]
+\whitespace trim
+
+<$let currentTiddler" value={{$:/language}} languageTitle={{!!name}}>
```

Comment:
There seems to be a typo .. `currentTiddler={{$:/language}}` let-widget has no value parameter

Reply:
Oh yes, sorry. Changed